sp800-63-3/sec6_xAL.md

```diff
@@ -176,9 +176,9 @@ The AAL selection does not mean the digital service provider will need to issue
 
 #### <a name="FAL_CYOA"></a> 6.3. Selecting FAL
 
-All FALs require assertions to have a baseline of protections, including signatures, expirations, audience restrictions, and others enumerated in [[SP 800-63C]](sp800-63c.html#assertions). When taken together, these measures make it so that assertions cannot be created or modified by an unauthorized party, and that a RP will not accept an assertion created for a different system. 
+All FALs require assertions to have a baseline of protections, including signatures, expirations, audience restrictions, and others enumerated in [[SP 800-63C]](sp800-63c.html#assertions). When taken together, these measures make it so that assertions cannot be created or modified by an unauthorized party, and that an RP will not accept an assertion created for a different system. 
```

Comment:
Oh no! I botched this conversation. When I said I agree with the general rule in that issue, I meant the rule (write it like you speak it), not the example given (which was wrong). It should be "an RP" because when you speak it you say "are pea," which starts with a vowel sound. We can catch these in next white glove, but for acronyms the pronunciation of the letter wins over the pronunciation of its expanded word.